### ISSUE TITLE
Use mod storage

### ISSUE THREAD

**Issue by numberZero** [octacian/digicompute#7]
Nowadays, Minetest provides a key-value storage for mods. It should be safer and more reliable than separate files.

**Comment by octacian** [comment]
Yes, I'm well aware of Minetest's modstorage - I've used it in numerous other mods since this. However, I don't intend to use it in digicompute. I have considered it in the past, but one of the key features of digicompute is that it is possible to write scripts for it outside of the game and then simply copy them directly into the filesystem of any computer, and vice versa.

This is important due to how the editor that I can provide through formspecs is not very intuitive, instead acting as an incentive to code outside of the game. Not only that, but on some systems (mine included) copy-paste between Minetest and other outside programs does not work, so I cannot simply code externally and then copy-paste it to Minetest.
